app/views/grids/_wikis.html.erb

```diff
@@ -1,58 +1,58 @@
-<div class="inline-wiki-container">
-<div style="max-height:500px;overflow:scroll;">
-<table class="table inline-grid <%= type %>-grid <%= className %> <%= className %>-<%= randomSeed %>">
-  <tr>
-    <th>Title</th>
-    <th>Updated</th>
-    <th>Version</th>
-    <th>Views</th>
-    <th>Likes<th>
-  </tr>
-  <% nodes.each do |wiki| %>
-    <tr>
-      <td style="width:40%">
-        <% if wiki.has_tag("pin:#{tagname}") %><i rel="tooltip" title="This item was pinned with the tag 'pin:<%= tagname %>'." class="fa fa-thumb-tack"></i> <% end %>
-        <i class="fa fa-file"></i> 
-        <a href="<%= wiki.path %>"><%= wiki.latest.title %></a>
-      </td>
-      <td><%= distance_of_time_in_words(Time.at(wiki.latest.created_at), Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %> <%= raw t('wiki._wikis.by') %> 
-      <a href="/profile/<%= wiki.latest.author.name %>"><%= wiki.latest.author.name %></a></td>
-      <td><%= wiki.revisions.length %></td>
-      <td><%= number_with_delimiter(wiki.views) %></td>
-      <td><%= number_with_delimiter(wiki.cached_likes) %></td>
-    </tr>
-  <% end %>
-</table>
-</div>
-<%= will_paginate @wikis, :renderer => BootstrapPagination::Rails if @paginated %>
-
-<a class="grid-embed grid-embed-<%= randomSeed %>" title="Embed this table on another site."><i class="fa fa-code" style="color:#aaa;"></i></a>
-
-<script>
-(function(){
-  $(".<%= className %>-<%= randomSeed %> .show-all a").click(function() {
-    $(".<%= className %>-<%= randomSeed %> tr.hide").toggleClass("hide");
-  });
-  $(".grid-embed-<%= randomSeed %>").click(function() {
-    prompt('Use this HTML code to embed this table on another site.', '<iframe style="border:none;" width="100%" height="900px" src="//publiclab.org/embed/grid/<% if type != "notes" %><%= type.gsub('questions', 'question').gsub('activities', 'activity') %>:<% end %><%= tagname %>"></iframe>')
-  });
-  setupGridSorters(".<%= className %>-<%= randomSeed %>");
-})()
-</script>
-
-<style>
-  a.grid-embed {
-    float:right;
-    padding:8px;
-  }
-  a.grid-embed:hover {
-    color: #888;
-  } 
-  th {
-    position: sticky;
-    top: 0;
-    z-index: 5;
-    background: #fff;
-}
-</style>
-</div>
+<div class="inline-wiki-container">
+<div style="max-height:500px;overflow:auto;">

```

Comment:
I believe the default value is `overflow:auto`. Could you try that out once? If that is the case, then maybe we can remove this part of the inline CSS entirely.

Reply:
@anirudhprabhakaran3 The default value for overflow is `visible`.
I tried removing the overflow property entirely, so there does not appear any scrollbar even on smaller screen sizes :

![image](https://user-images.githubusercontent.com/78212650/156918679-5608463e-b685-4bc3-9755-f2648310fef0.png)


And with `overflow:auto`, on smaller screen sizes, a scrollbar appears as and when needed:

![image](https://user-images.githubusercontent.com/78212650/156918668-60db0bb2-6c6a-4318-b35b-c2d7f9500c00.png)


Which one looks better? Or maybe, I could also add a custom scrollbar whenever it appears (i.e. with `overflow:auto`)?

Reply:
Aah i see that is my mistake. Yeah, overflow:auto looks good and I think that's what we were trying to do here. Looks good!

Reply:
Thanks :)